tests/plugins/wysiwygarea/manual/detached.html

```diff
@@ -0,0 +1,48 @@
+<button id="toggleButton1">Toggle</button>
+<button id="setButton">Set data</button>
+
+<div>
+	<div id="editor">
+		<p>Lorem ipsum dolor sit amet.</p>
+	</div>
+</div>
+
+
+<div id="fakedId"></div>
+<div id="fakedId1"></div>
+
+
+<script>
+	var editorElement = document.getElementById("editor");
```

Comment:
Inside `<script>` tags we use [single quotes](https://github.com/ckeditor/ckeditor4/blob/major/dev/docs/codestyle.md#224-quotes) (`'`) instead of double quotes. That is also relevant for other lines like 18, 30, 31 etc.

Reply:
We are also missing the spaces after starting bracket and before closing one. Please fix it throughout the whole file.